CorsixTH/Lua/diseases/serious_radiation.lua

```diff
@@ -43,13 +43,16 @@ disease.initPatient = function(patient)
   patient:setLayer(1, math.random(0, 3) * 2)
   patient:setLayer(3, 0)
   patient:setLayer(4, 0)
-  patient.th:setEffect(AnimationEffect.Glowing)
+  patient.th:setPatientEffect(AnimationEffect.Glowing)
 end
-disease.updatePatient = function(patient)
+disease.treatPatient = function(patient)
+  patient.th:setPatientEffect(AnimationEffect.None)
+end
+disease.afterLoad = function(patient)
```

Comment:
I don't have a good recommendation yet, but I don't like this.
afterLoad conventionally takes two parameters old and new version; that pattern applies everywhere else in CorsixTH.

Perhaps the problem is disease is suppose to be a collection of values and it's becoming increasingly a class with logic and side effects. Maybe diseases should just have an effect, and patient should apply it as appropriate...

Reply:
That is closer to how Type is managed

Reply:
Yeah, I didn't like it either. Thanks for explaining the parameters, I was searching the code to try to figure out they were.

I like your suggestion to just have an effect on the disease that the patient class applies, I've switched to this strategy now and tested / verified it's working.